listenbrainz/webserver/views/user.py

```diff
@@ -295,70 +288,18 @@ def recommendation_playlists(user_name: str):
     props = {
         "playlists": playlists,
         "user": user_data,
-        "active_section": "recommendations",
         "playlist_count": playlist_count,
         "logged_in_user_follows_user": logged_in_user_follows_user(user),
     }
 
     return render_template(
-        "playlists/playlists.html",
+        "playlists/recommendations.html",
         active_section="recommendations",
         props=ujson.dumps(props),
         user=user
     )
 
 
-@user_bp.route("/<user_name>/collaborations/")
-@web_listenstore_needed
-def collaborations(user_name: str):
-    """ Show playlists a user collaborates on """
-
-    offset = request.args.get('offset', 0)
-    try:
-        offset = int(offset)
-    except ValueError:
-        raise BadRequest("Incorrect int argument offset: %s" %
-                         request.args.get("offset"))
-
-    count = request.args.get("count", DEFAULT_NUMBER_OF_PLAYLISTS_PER_CALL)
-    try:
-        count = int(count)
-    except ValueError:
-        raise BadRequest("Incorrect int argument count: %s" %
-                         request.args.get("count"))
-
-    user = _get_user(user_name)
-    user_data = {
-        "name": user.musicbrainz_id,
-        "id": user.id,
-    }
-
-    include_private = current_user.is_authenticated and current_user.id == user.id
```

Comment:
This line was not re-added and will cause failures.

Reply:
It's in there, just hidden behind code folding:
https://github.com/metabrainz/listenbrainz-server/pull/2344/files#diff-e91083bcd314135e0a5ef06f4ab7e4121bf1f2ad9360b570e1e2dd6033a75bf3R220

Reply:
Shit, totally missed the fold. Sorry.